src/style-breadcrumb.scss

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2022 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro tokens
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+
+@import './../node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints';
+@import './../node_modules/@alaskaairux/webcorestylesheets/dist/core';
+
+/* stylelint-disable selector-no-qualifying-type, declaration-empty-line-before,  order/properties-order */
+
+:host {
+  display: inline-flex;
+
+  .hyperlink {
+    &:focus,
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  ::slotted(auro-icon[name='chevron-left']) {
+    display: none;
+  }
+
+  // Inserts the right chevron icon after each breadcrumb
+  &:after {
+    position: relative;
+    display: inline-block;
+
+    width: 24px;
+    height: 24px;
+
+    top: -1px;
+    line-height: 24px;
+
+    margin-right: 5px;
+    margin-left: var(--auro-size-100);
+
+    background-color: var(--auro-color-icon-primary-on-light);
+    content: '';
+
+    -webkit-mask-box-image: url('https://unpkg.com/@alaskaairux/icons@latest/dist/icons/interface/chevron-right.svg');
+    mask-image: url('https://unpkg.com/@alaskaairux/icons@latest/dist/icons/interface/chevron-right.svg');
+    mask-repeat: no-repeat;
+  }
+
+  a {
+    color: var(--auro-color-ui-default-on-light);
+    text-decoration: none;
+  }
+}
+
+// Hides the home icon for all breadcrumbs besides the first
+:host(:not(:first-of-type)) {
+  ::slotted(auro-icon[name='home-stroke']) {
+    display: none;
+  }
+}
+
+// Last breadcrumb should act as a disabled breadcrumb
+:host(:last-of-type) {
+  color: var(--auro-color-text-secondary-on-light);
+
+  // Hides last breadcrumb divider
+  &:after {
+    content: unset;
+  }
+}
+
+@media (max-width: 768px) {
```

Comment:
This is not a breakpoint spec. Where did this come from? 

See mixin support here - https://alaskaairlines.github.io/WebCoreStyleSheets/#mixin-auro_breakpoint--sm

Token name | Value 
-- | --
var(--auro-breakpoint-sm) | 660px 
var(--auro-breakpoint-md) | 1024px 
var(--auro-breakpoint-lg) | 1232px